### PR TITLE
add doc on cross-arch builds

### PR DESCRIPTION
**Pull request by sallyom**
Add documentation for running cross-architecture bib

I might have used too many words :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,18 +105,20 @@ Usage:
     <imgref>
 
 Flags:
-      --config string   build config file
-      --tls-verify      require HTTPS and verify certificates when contacting registries (default true)
-      --type string     image type to build [qcow2, ami] (default "qcow2")
+      --config string          build config file
+      --tls-verify             require HTTPS and verify certificates when contacting registries (default true)
+      --type string            image type to build [qcow2, ami] (default "qcow2")
+      --target-arch string     architecture to build image for (default is the native architecture)
```
Comment (cgwalters) [inline README.md:111]: (Not a new issue but it's unclear to me why we need to duplicate this table)

**Review by cgwalters — DISMISSED** [review]

**Review by mvo5 — DISMISSED** [review]
Thank you! And sorry that this review took forever, it somehow slipped through.

File: README.md [hunk 2]
```diff
@@ -130,6 +132,17 @@ The following image types are currently available via the `--type` argument:
 | `qcow2` **(default)** | [QEMU](https://www.qemu.org/)                                                         |
 | `iso`                 | An unattended Anaconda installer that installs to the first disk found.               |
 
+## 💾 Target architecture
+
+Specify the target architecture of the system on which the disk image will be installed on. By default,
```
Comment (mvo5) [inline README.md:137]: I wonder if we should slip in the word "experimental" here somewhere but then we print it on the commandline and while this needs more real world testing we are somewhat confident that the feature will work fine :)

**Review by mvo5 — APPROVED** [review]
I rebased this and fixed the conflicts. Sorry that this has not landed yet, I will try to get it in now.

File: README.md (hunk 2 above)
Comment (mvo5) [reply README.md:172]: Nevermind, this is no longer experimental it now used for a while.

**Pull request merged**